### PR TITLE
attempt at inline figure

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

File: src/scss/blocks/_prose.scss [hunk 1]
```diff
@@ -6,10 +6,7 @@
 
   --flow-space: #{get-space('size-1')};
 
-  > * {
-    margin-inline: auto;
-    max-width: $global-prose-element-max-width;
-  }
+  max-width: $global-prose-element-max-width;
```
Comment (hankchizljaw) [inline src/scss/blocks/_prose.scss:9]: We're doing to need a `display: flow-root` here too to clear any floats 👍

File: src/scss/blocks/_figure.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+figure[data-float] {
+  max-width: calc(#{$global-prose-element-max-width} / 2);
+  margin-bottom: var(--flow-space, 1em);
+}
+
+figure[data-float="left"] {
+  float: left;
+  margin-right: var(--flow-space, 1em);
```
Comment (hankchizljaw) [inline src/scss/blocks/_figure.scss:16]: Let's replace this with `margin-inline-end: get-space('size-1')`

File: src/scss/blocks/_figure.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+figure[data-float="right"] {
+  float: right;
+  margin-left: var(--flow-space, 1em);
```
Comment (hankchizljaw) [inline src/scss/blocks/_figure.scss:21]: Let's replace this with `margin-inline-start: get-space('size-1')`

**Review by hankchizljaw — CHANGES_REQUESTED** [review]
Just a couple of nits, but LGTM otherwise

**Comment by devnook** [comment]
ptal

**Review by hankchizljaw — APPROVED** [review]
LGTM. Nice one, @devnook!

**Pull request merged**